public/_redirects

```diff
@@ -1,2 +1,2 @@
-/config /#/config 302
+/config /#/ipfs-sw-config 302
```

Comment:
the redirect happens only if it hits the ipfs hosted site backend.

Reply:
With that said though, we probably don't need redirect from /config to anything anymore since we're not doing any requests to /config anymore.

`helia-sw` is still needed as a catch all